spread/project.go

```diff
@@ -125,6 +125,11 @@ type System struct {
 	// Only for Google so far.
 	SecureBoot bool `yaml:"secure-boot"`
 
+	// Request a specific CPU family, e.g. "Intel Skylake". This follows
+	// the GCE naming today:
```

Comment:
This comment needs updating, it's only following the naming for GCE when it's working with GCE but on AWS it would be different so this needs to be more generic.